tests/sles4sap/redirection_tests/ensa2_schedule_sapinstance_tests.pm

```diff
@@ -61,6 +74,8 @@ sub run {
     # Schedule all tests using `loadtest` call
     for my $test_name (
         'Kill_sapinstance_ASCS',
```

Comment:
why having the 4 keys repeated here too? why not to iterate over all the keys in `%scenarios` ?